css/main.css

```diff
@@ -0,0 +1,611 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
```

Comment:
### Optional 

I'll mark this as a suggestion rather than a requirement since the last code reviewer missed it so you can implement it if you wish!

Please make your code as DRY as it can be. You are repeating justify-content property with center value in multiple lines. You could make a class and reuse it instead of having multiple instances 🧐